packages/css-library/tokens/spacing.json

```diff
@@ -0,0 +1,37 @@
+{
+  "units": {
+    "0": { "value": "0"},
+    "1px": { "value": "1px"},
+    "0p25": { "value": "2px"},
+    "0p5": { "value": "4px"},
+    "1": { "value": "8px"},
+    "1p5": { "value": "12px"},
+    "2": { "value": "16px"},
+    "2p5": { "value": "20px"},
+    "3": { "value": "24px"},
+    "4": { "value": "32px"},
+    "5": { "value": "40px"},
+    "6": { "value": "48px"},
+    "7": { "value": "56px"},
+    "8": { "value": "64px"},
+    "9": { "value": "72px"},
+    "10": { "value": "80px"},
+    "15": { "value": "120px"},
+    "neg-1px": { "value": "-{units.1px}"},
+    "neg-0p25": { "value": "-{units.0p25}"},
+    "neg-0p5": { "value": "-{units.0p5}"},
+    "neg-1": { "value": "-{units.1}"},
+    "neg-1p5": { "value": "-{units.1p5}"},
+    "neg-2": { "value": "-{units.2}"},
+    "neg-2p5": { "value": "-{units.2p5}"},
+    "neg-3": { "value": "-{units.3}"},
+    "neg-4": { "value": "-{units.4}"},
+    "neg-5": { "value": "-{units.5}"},
+    "neg-6": { "value": "-{units.6}"},
+    "neg-7": { "value": "-{units.7}"},
+    "neg-8": { "value": "-{units.8}"},
+    "neg-9": { "value": "-{units.9}"},
+    "neg-10": { "value": "-{units.10}"},
+    "neg-15": { "value": "-{units.15}"}
+ }
```

Comment:
These are all pixel based [like USWDS](https://designsystem.digital.gov/design-tokens/spacing-units/), but [in `formation` we use `rem`s](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/40dbe6725a106534e08fe6b8e6d03a946a82986e/packages/formation/sass/base/_b-variables.scss#L206-L241).

Reply:
USWDS is a mix of px and rem: 
"The documentation below displays a target value in pixels, but the final output for tokens above 2px and below -2px is the eqivalent value in rem, which will vary depending on your project settings."

Reply:
Alright, I've made some changes. The base rem unit is 10px, so this does make the conversion a little wonky.